examples/canjs/.gitignore

```diff
@@ -0,0 +1,7 @@
+node_modules
+!node_modules/canjs-localstorage/can.localstorage.js
+!node_modules/canjs/can.jquery.js
+!node_modules/jquery/dist/jquery.js
+!node_modules/todomvc-common/base.js
+!node_modules/todomvc-app-css/index.css
+!node_modules/todomvc-common/base.css
```

Comment:
yep and it makes this process insane hahaha :finnadie:  look at the canjs ignore

```
    8 node_modules/canjs/steal/*                                                                                                                                                                                                                                                                                                                                          |
    9 node_modules/canjs/amd-dev/*                                                                                                                                                                                                                                                                                                                                        |
   10 node_modules/canjs/amd/*                                                                                                                                                                                                                                                                                                                                            |
_  11 !node_modules/canjs/amd/can.js                                                                                                                                                                                                                                                                                                                                      |
   12 node_modules/canjs/bower.json                                                                                                                                                                                                                                                                                                                                       |
   13 node_modules/canjs/can.construct.proxy.js                                                                                                                                                                                                                                                                                                                           |
   14 node_modules/canjs/can.construct.super.js                                                                                                                                                                                                                                                                                                                           |
   15 node_modules/canjs/can.control.plugin.js                                                                                                                                                                                                                                                                                                                            |
   16 node_modules/canjs/can.dojo.dev.js                                                                                                                                                                                                                                                                                                                                  |
   17 node_modules/canjs/can.dojo.js                                                                                                                                                                                                                                                                                                                                      |
   18 node_modules/canjs/can.dojo.min.js                                                                                                                                                                                                                                                                                                                                  |
   19 node_modules/canjs/can.ejs.js                                                                                                                                                                                                                                                                                                                                       |
   20 node_modules/canjs/can.fixture.js                                                                                                                                                                                                                                                                                                                                   |
   21 node_modules/canjs/can.jquery.dev.js                                                                                                                                                                                                                                                                                                                                |
   22 node_modules/canjs/can.jquery.js                                                                                                                                                                                                                                                                                                                                    |
   23 node_modules/canjs/can.jquery.min.js                                                                                                                                                                                                                                                                                                                                |
   24 node_modules/canjs/can.list.sort.js                                                                                                                                                                                                                                                                                                                                 |
   25 node_modules/canjs/can.map.attributes.js                                                                                                                                                                                                                                                                                                                            |
   26 node_modules/canjs/can.map.backup.js                                                                                                                                                                                                                                                                                                                                |
   27 node_modules/canjs/can.map.delegate.js                                                                                                                                                                                                                                                                                                                              |
   28 node_modules/canjs/can.map.list.js                                                                                                                                                                                                                                                                                                                                  |
   29 node_modules/canjs/can.map.setter.js                                                                                                                                                                                                                                                                                                                                |
   30 node_modules/canjs/can.map.validations.js                                                                                                                                                                                                                                                                                                                           |
   31 node_modules/canjs/can.model.queue.js                                                                                                                                                                                                                                                                                                                               |
   32 node_modules/canjs/can.mootools.dev.js                                                                                                                                                                                                                                                                                                                              |
   33 node_modules/canjs/can.mootools.js                                                                                                                                                                                                                                                                                                                                  |
   34 node_modules/canjs/can.mootools.min.js                                                                                                                                                                                                                                                                                                                              |
   35 node_modules/canjs/can.object.js                                                                                                                                                                                                                                                                                                                                    |
   36 node_modules/canjs/can.route.pushstate.js                                                                                                                                                                                                                                                                                                                           |
   37 node_modules/canjs/can.view.modifiers.js                                                                                                                                                                                                                                                                                                                            |
   38 node_modules/canjs/can.yui.dev.js                                                                                                                                                                                                                                                                                                                                   |
   39 node_modules/canjs/can.yui.js                                                                                                                                                                                                                                                                                                                                       |
   40 node_modules/canjs/can.yui.min.js                                                                                                                                                                                                                                                                                                                                   |
   41 node_modules/canjs/can.zepto.dev.js                                                                                                                                                                                                                                                                                                                                 |
   42 node_modules/canjs/can.zepto.js                                                                                                                                                                                                                                                                                                                                     |
   43 node_modules/canjs/can.zepto.min.js                                                                                                                                                                                                                                                                                                                                 |
   44 node_modules/canjs/package.json                                                                                                                                                                                                                                                                                                                                     |
   45 node_modules/canjs/readme.md
```